docs/book/guides/low-level-api/chapter-1.md

```diff
@@ -4,7 +4,7 @@ description: Create your  first step.
 
 If you want to see the code for this chapter of the guide, head over to the [GitHub](https://github.com/zenml-io/zenml/tree/main/examples/low_level_guide/chapter_1.py).
 
-# Chapter 1: Create an importer step to load data
+# Create an importer step to load data
```

Comment:
Im impartial to adding that but did you feel like it looks better this way or was this automatically taken out somehow? :-)

Reply:
I assumed it was a mistake. It looked weird to have it twice on the same page, with nothing really in between those two lines. We also don't really do that kind of double headline within our docs, at least not too often, so it felt the H1 would already stand out, so leave the H2 text on its own.

Reply:
Okay makes sense!